qlib/SftpPoller.qm

```diff
@@ -143,6 +143,8 @@ public namespace SftpPoller {
                 "reopts": 0,
                 "timeout": 30s,
                 "binary": False,
+                "writable" : False,
```

Comment:
please remove the space before the colon (make the style like the other declarations above)